examples/convection/exponential1D/mesh1D.py

```diff
@@ -17,7 +17,8 @@
 
 We define a 1D mesh
 
-.. index:: Grid1D
+.. index::
+   single: Grid1D
```

Comment:
What is this supposed to do? Is something supposed to link from the General Index to this reference? I can't find that in the index. 

BTW, in my version of the document build the Global Module Index is a dead link from the the main index.html. That's the same on the version hosted on CTCMS.

Reply:
`.. index:: Grid1D` is a shorthand that napoleon doesn't understand and I didn't feel like fighting with. I think it may be a [sphinx augmentation](http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#index-generating-markup).

I see what you mean about the Global Module Index. I'll look into that. Search is also broken, and has been for awhile. That seems to be an [issue with sphinx](https://github.com/sphinx-doc/sphinx/issues/5460) that may or may not be fixed in the version available for Python 2.